tooling/templatize/pkg/pipeline/types.go

```diff
@@ -0,0 +1,24 @@
+package pipeline
+
+type Pipeline struct {
+	RolloutName string  `yaml:"rolloutName"`
+	Steps       []*Step `yaml:"steps"`
+}
+
+type Step struct {
+	Name          string `yaml:"name"`
+	Subscription  string `yaml:"subscription"`
+	ResourceGroup string `yaml:"resourceGroup"`
+	Action        Action `yaml:"action"`
+}
+
+type Action struct {
+	Type      string   `yaml:"type"`
+	Command   string   `yaml:"command"`
+	Variables []EnvVar `yaml:"variables"`
```

Comment:
My hunch is it would be much more ergonomic to have these in the `Makefile`

Reply:
agreed. let's keep it like this for the time being so we can move on to something that works now.
there is a way forward to not require this section at all.